src/products/views/ProductList/ProductList.tsx

```diff
@@ -215,9 +219,18 @@ export const ProductList: React.FC<ProductListProps> = ({ params }) => {
     }
   );
 
+  const columnIdsFilter = columns => {
+    columns = columns.filter(column => isAttributeColumnValue(column));
+    columns.forEach(
+      (attribute, index, arr) =>
+        (arr[index] = getAttributeIdFromColumnValue(attribute))
+    );
+    return columns;
+  };
```

Comment:
I think this can be shortened to

```
  function filterColumnIds(columns: ProductListColumns[]) {
    return columns.filter(isAttributeColumnValue).map(getAttributeIdFromColumnValue)
  }
```

Reply:
cool, working on it. Thanks

Reply:
Also, we could move it to the util library to be able to reuse it in other places as we need it.

Reply:
Never mind, it's specific for the available query in that view. I updated the description to include issue #478 created today.

Done, please review it again and thank you for the feedback :)